packages/discord.js/src/managers/GuildBanManager.js

```diff
@@ -199,6 +199,40 @@ class GuildBanManager extends CachedManager {
     await this.client.rest.delete(Routes.guildBan(this.guild.id, id), { reason });
     return this.client.users.resolve(user);
   }
+
+  /**
+   * Ban up to 200 users from a guild, and optionally delete previous messages sent by the banned users.
+   * @param {Collection<Snowflake, User>|UserResolvable[]} users The users to ban
+   * @param {BanBulkOptions} options The options for bulk banning users
+   * @returns {Promise<BanBulkResult>} Returns an object with `bannedUsers` key containing the IDs of the banned users
+   * and the key `failedUsers` with the IDs that could not be banned or were already banned.
+   * @example
+   * // Bulk ban users by ids (or with user/guild member objects)
+   * guild.bans.bulkCreate(['84484653687267328'], { deleteMessageSeconds: 7 * 24 * 60 * 60 })
+   *   .then(result => {
+   *     console.log(`Banned ${result.bannedUsers.length} users, failed to ban ${result.failedUsers.length} users.`)
+   *   })
+   *   .catch(console.error);
+   */
+  async bulkCreate(users, options) {
```

Comment:
Let's make options optional, no reason to have to pass in an object and leave it empty